pkg/auth/authproviders/openshift/internal/connector/connector.go

```diff
@@ -0,0 +1,67 @@
+package connector
```

Comment:
Let's just move this to `dexconnector` as a separate file, then we have all code copied from the dex dep in a single place / package.